department-of-veterans-affairs/va.gov-cms
# VACMS-4443_4444 vc mobile & outstation

## Description
See #4443 #4444

## Testing done
Visual, behat, phpunit

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/109189832-7518ec00-7762-11eb-876d-65cd19990aa2.png)
![image](https://user-images.githubusercontent.com/2404547/109189850-7a763680-7762-11eb-95a1-b16b03d5baf9.png)
![image](https://user-images.githubusercontent.com/2404547/109189860-7e09bd80-7762-11eb-89d5-238bba95d018.png)
![image](https://user-images.githubusercontent.com/2404547/109189875-8104ae00-7762-11eb-8d7e-e9e34f342b14.png)

## QA steps
- As an administrator, go to `node/add/vet_center_mobile_vet_center` and visually verify form fields match form screenshot above.
- Save form, visually verify node view matches node view screenshot above.
- As an administrator, go to `node/add/vet_center_outstation` and visually verify form fields match form screenshot above.
- Save form, visually verify node view matches node view screenshot above.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
